modules/tensor_mechanics/test/tests/dynamics/time_integration/tests

```diff
@@ -53,14 +49,23 @@
     input = 'hht_test_ti.i'
     exodiff = 'hht_test_out.e'
     abs_zero = 1e-09
-    compiler = 'GCC CLANG'
 
     requirement = "The mechanics system shall accurately predict the dynamic response of a linear "
                   "elastic system using Hilber-Hughes-Taylor (HHT) time integration when velocity "
                   "and acceleration of the system are calculated using the Newmark-Beta time "
                   "integrator."
     design = "Dynamics.md"
     issues = "#12185"
-    prereq = "hht"
+  []
+
+  [ad_newmark_action]
+    type = 'Exodiff'
+    input = 'ad_newmark_action.i'
+    exodiff = 'ad_newmark_action_out.e'
+    abs_zero = 1e-09
+
+    requirement = "The dynamic tensor mechanics action shall support automatic differentiation."
```

Comment:
Should the requirement mention the Newmark integrator since it's in the time integrator folder? I assume we cannot make claims on verification for this particular test.